modules/nw-rfhe-creating_bmc_event_sub.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+
+:_content-type: PROCEDURE
+[id="nw-rfhe-creating_bmc_event_sub_{context}"]
+= Subscribing to Redfish hardware events
+
+You can configure the baseboard management controller (BMC) to send Redfish hardware events to subscribed applications running in {product-title} cluster. Example Redfish events include an increase in device temperature, or removal of a device. Applications subscribe to Redfish hardware events using a REST API.
+
+[IMPORTANT]
+====
+You can only create a `BMCEventSubscription` CR for physical hardware that supports Redfish and has a vendor interface set to `redfish` or `idrac-redfish`.
+====
+
+[NOTE]
+====
+Use the `BMCEventSubscription` CR to subscribe to pre-defined Redfish events. The Redfish standard does not provide an option to create specific alerts and thresholds. For example, to receive an alert event when an enclosure's temperature exceeds 40° Celsius, you need to manually configure the event according to the vendor's recommendations.
+====
+
+To subscribe to Redfish hardware events for the node using a `BMCEventSubscription` custom resource (CR), perform the following procedure:
```

Comment:
```suggestion
Perform the following procedure to subscribe to Redfish hardware events for the node using a `BMCEventSubscription` custom resource (CR) .
```
Suggestion for rewording to avoid using : at the end of the sentence.

Reply:
applied